ckanext/ontario_theme/i18n/fr/LC_MESSAGES/ckanext-ontario_theme.po

```diff
@@ -2488,6 +2488,10 @@ msgstr "Servez-vous de l’API de données"
 msgid "CKAN Data API calls"
 msgstr "appels de l’API de données CKAN"
 
+#: ckanext/ontario_theme/templates/internal/ajax_snippets/api_info.html:129
+msgid "A simple request to the data API using Python."
+msgstr "Une requête HTTP à l’API des données en utilisant Python."
```

Comment:
Is simple supposed to be in the French translation, or is it okay without?

Reply:
Yes let's do it for consistency